python/IECoreDelightTest/RendererTest.py

```diff
@@ -101,6 +102,43 @@ def testOutput( self ) :
 
 		self.assertTrue( ( self.temporaryDirectory() / "beauty.exr" ).exists() )
 
+	def testOutputMultipart( self ) :
+
+		r = GafferScene.Private.IECoreScenePreview.Renderer.create(
+			"3Delight",
+			GafferScene.Private.IECoreScenePreview.Renderer.RenderType.Batch
+		)
+
+		r.output(
+			"layer_a",
+			IECoreScene.Output(
+				str( self.temporaryDirectory() / "multipart.exr" ),
+				"exr",
+				"rgba"
+			)
+		)
+
+		r.output(
+			"layer_b",
+			IECoreScene.Output(
+				str( self.temporaryDirectory() / "multipart.exr" ),
+				"exr",
+				"color shader:diffuse"
+			)
+		)
+
+		r.render()
+		del r
+
+		self.assertTrue( ( self.temporaryDirectory() / "multipart.exr" ).exists() )
+
+		i = IECoreImage.ImageReader( os.path.join( self.temporaryDirectory(), "multipart.exr" ) )
+		h = i.readHeader()
+		subimages = h['oiio:subimages']
+		del i
```

Comment:
It's not essential that you change this now, but we're close to phasing out `IECoreImage.ImageReader` everywhere, so if you have time it'd be great if this check could be performed with the `OpenImageIO` module directly instead.

Reply:
Got it! Absolutely no problem to switch the test to the `OpenImageIO` module.